eng/pipelines/coreclr/templates/build-perf-maui-apps-net7.yml

```diff
@@ -119,11 +119,12 @@ steps:
   - script: |
       curl -o NuGet.config 'https://raw.githubusercontent.com/dotnet/maui/main/NuGet.config'
       curl -o dotnet-install.sh 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh'
+      curl -Lo maui-version.json 'https://aka.ms/dotnet/sdk/maui/net7.0.json'
+      version=$(sed -nr 's/\s*"version": "(.*)"/\1/p' ./maui-version.json)
```

Comment:
Doesn't this need to be `export version=..`?

Reply:
Clarified the name of the json file, as it contains the supported sdk version rather than the maui-version. As such, we also do not need the export for the variable since we use it right away in the same script task and not outside of it.